fava/templates/macros/_account_macros.html

```diff
@@ -69,3 +69,20 @@ <h1 class="droptarget" data-account-name="{{ account_name }}">
     {% endif %}
 </h1>
 {% endmacro %}
+
+{% macro balance_with_budget(currency, number, budget=None, show_currency=False, css_class="balance") %}
+<span class="{{ css_class }}">
+    {% if number %}
+        {% if budget %}
+            {% set budget = (budget - number)|round(10) %}
+            {% if budget == 0.0 %}
+                {% set budget = (budget - budget) %}  {# we have to round budget and use the weird (budget-budget) trick to make it really zero, not near zero #}
```

Comment:
`budget` is a `Decimal`, and `budget = (budget - number)` does not return an exact `0.0`. It passes the `if budget == 0.0`, but then it is not really zero, and so the template-`{% if budget > 0.0 %}` always is greater than 0.0.

I could not find another, more elegant way to make this work.